fighthealthinsurance/medicaid_gov_api.py

```
@@ -0,0 +1,453 @@
+    * No document type declaration. Sitemaps have no DTD, so a ``<!DOCTYPE``
+      or ``<!ENTITY`` here is either a mistake or an entity-expansion attempt.
+      Current expat refuses external entities and is no longer vulnerable to
+      the classic expansion attacks, but rejecting the declaration outright
+      costs nothing and doesn't depend on the parser version underneath us.
+    """
+    if len(content) > _MAX_SITEMAP_BYTES:
+        raise ValueError(f"sitemap document too large ({len(content)} bytes)")
+    head = content[:2048].lower()
+    if b"<!doctype" in head or b"<!entity" in head:
+        raise ValueError("sitemap document carries a doctype/entity declaration")
+    return ET.fromstring(content)
```

Ruff (0.16.2)

[error] Using `xml` to parse untrusted data is known to be vulnerable to XML attacks; use `defusedxml` equivalents

(S314)